### ISSUE TITLE
[Legitimate Site Blocked] localtrade.cc

### ISSUE THREAD

**Issue by tabletop74** [MetaMask/eth-phishing-detect#88637]
https://localtrade.cc/

**Comment by tabletop74** [comment]
Hello once again! You've closed our previous ticket and not responeded.
Here are links to our social media resourses:
Telegram https://t.me/localtradecc
Twitter https://twitter.com/LocaltradeC
Whitepaper https://localtrade.cc/LocalTrade-Whitepaper-2.0-ENG.pdf

You've said that "twitter hasn't been active since early 2023." and closed the ticket - this is unacceptable in a business dialogue.

Once again we ask to unblock our website. Exchange is working absolutely fine and what is the reason blocking it if it doesn't even have wallet connect feature. On current stage the project is on hold while preparing new marketing campaign. 

Waiting for your response.

**Comment by AlexHerman1** [comment]
Hi,

Is the management team of this exchange doxxed? Where is the business incorporated and what countries does it serve?
